modules/task/execution.go

```diff
@@ -37,14 +37,14 @@ type fnRegister struct {
 
 var fnLookup = fnRegister{fnNameMap: make(map[string]interface{})}
 
-// fnSignature represents a function and its arguments
-type fnSignature struct {
+// FnSignature represents a function and its arguments
+type FnSignature struct {
```

Comment:
FnSignature and Backend are different things. I am exposing FnSignature so I can create test function structs in uberfx-private or end to end tests.